Shared/UI/Views/View.swift

```diff
@@ -30,9 +30,14 @@ class View: UIView {
 
     required init?(coder aDecoder: NSCoder) {
         super.init(coder: aDecoder)
-        self.initializeSubviews()
     }
 
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+        self.initializeSubviews()
+    }
```

Comment:
You can't call initialize views from init with coder because the outlets aren't hooked up at that time.

Reply:
nice